modules/mobile/app/models/mobile/v0/adapters/lighthouse_claims_overview.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative 'claims_overview'
+
+module Mobile
+  module V0
+    module Adapters
+      class LighthouseClaimsOverview < ClaimsOverview
```

Comment:
subclassing the original claimsOverview since we still need all the same logic to process appeals as that's not changing currently.